smart-storage-unit/packages/contracts/mud.config.ts

```diff
@@ -12,6 +12,13 @@ export default defineWorld({
         ratioOut: "uint64",
       },
       key: ["smartObjectId", "itemIn"],
+    },
+    DAppConfig: {
```

Comment:
Its not a good pattern to store configs on chain.

Reply:
Based on the functionality, table name can be changed perhaps

Reply:
I agree, DAppConfig should have better naming. This table name is misleading and suggested different purpose than what it actually does